src/components/pages/AdminDashboard/DevTools/DayComponents/01_Sat-Mon.js

```diff
@@ -0,0 +1,112 @@
+import React, { useEffect } from 'react';
+import { connect } from 'react-redux';
+import { useHistory } from 'react-router-dom';
+import { Layout, Button, Radio } from 'antd';
+
+import { tasks } from '../../../../../state/actions/index';
+import { date } from '../../../../../state/actions/index';
+
+const { Header, Content, Footer } = Layout;
+
+const SatMon = ({ setDate }) => {
+  const { push } = useHistory();
+
+  const adminDash = () => {
+    push('/admin');
+  };
+
+  const gameStageUrl = '/child/mission-control';
+
+  const findDayOfWeekReference = 1;
+  // Saturday was 6
+  // Sunday was 0
+  // Monday was 1
+
+  const handleSim = () => {
+    push(`${gameStageUrl}`);
+  };
+
+  const findNextDayOfWeek = selectedDay => {
```

Comment:
Since we call this function for each of the day components, we can likely pull it out into a hook to implement into each component -- that way we don't have to repeat the whole function in each component